doc/internal/man3/DEFINE_SPARSE_ARRAY_OF.pod

```diff
@@ -0,0 +1,111 @@
+=pod
+
+=head1 NAME
+
+DEFINE_SPARSE_ARRAY_OF, sa_TYPE_new, sa_TYPE_free, sa_TYPE_free_leaves,
+sa_TYPE_num, sa_TYPE_doall, sa_TYPE_doall_arg, sa_TYPE_get, sa_TYPE_set
+- sparse array container
+
+=head1 SYNOPSIS
+
+=for comment generic
```

Comment:
Oh interesting! What happens if you remove this line?
(I actually think it would be a good idea for the doc-nuts check to look at the internal docs as well... I don't think it will work without tweaks, but could be worth it)

Reply:
Seemingly nothing.  There are over a dozen other .pod files with this line.

Reply:
Yeah I know. We should review the need some day...

Reply:
Ah, now I remember, that comment is for man pages where the file name doesn't correspond to any of the names in the NAME section, i.e. a hint to doc-nits to shut up about that sort of discrepancy.  Since this manual does have a file name that corresponds to one of the NAMEs, it's harmless to remove this line, yeah.